spec/std/isa/ext/V.yaml

```
@@ -15,6 +15,212 @@ versions:
 description: |
   TODO
 params:
+  ELEN:
+    description: |
+      The maximum size in bits of a vector element that any operation can produce or consume, _ELEN_ {ge} 8, which
+      must be a power of 2.
+    schema:
+      type: integer
+      enum: [8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536]
+  VLEN:
+    description: |
+      The number of bits in a single vector register, _VLEN_ {ge} ELEN, which must be a power of 2, and must be no greater than 2^16^.
+    schema:
+      type: integer
+      enum: [8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536]
+  SEWmin:
```

We may want to observe an invariant that all params consist only of upper case and underscore. Looking at the schema, though, I think we already do.
```suggestion
  SEW_MIN:
```